cockatrice/src/window_main.cpp

```diff
@@ -337,7 +340,7 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
             formattedMissingFeatures = "Missing Features: ";
             for (int i = 0; i < missingFeatures.size(); ++i)
                 formattedMissingFeatures.append(QString("\n     %1").arg(QChar(0x2022)) + " " + missingFeatures.value(i)   );
-            formattedMissingFeatures.append("\nTo update your client, go to Help -> Check for Updates.");
+            formattedMissingFeatures.append(QString("<br>") + tr("To update your client, go to \"Help -> Check for Client Updates\"."));
```

Comment:
This message was using `\n` to avoid its text being rendered as html.
Since the popup contains text coming from the server, rendering it as html could be problematic if you don-t escape it. Think of a malicious server admin adding a new bogus "feature" using html to inject code into user's popups.
Please have a look at http://doc.qt.io/qt-5.8/qmessagebox.html#rich-text-and-the-text-format-property